### ISSUE TITLE
Corrupt Excel file created for campaign export

### ISSUE THREAD

**Issue by nullpainter** [securestate/king-phisher#227]
## Issue Description
When exporting a campaign to Excel, Microsoft Excel complains that the file contains corrupted content. Accepting the proposed repairs appears to renders the spreadsheet correctly.

## Reproduction Steps
1. Start King Phisher
2. File -> Export -> Campaign -> Excel
3. Open the XLSX file in Excel

## Environment Details

Host OS: Windows 7
King Phisher Version: 1.8.0
Microsoft Excel version: 2010

![image](https://user-images.githubusercontent.com/1732347/28193770-7ca79f84-6893-11e7-9d02-5383b468a886.png)

**Comment by zeroSteiner** [comment]
Have you tried opening the same file in a newer version of Excel? I suspect it has something to do with it being Excel 2010.

**Comment by nullpainter** [comment]
I don't have a newer version of Excel available unfortunately, so am unable to be of much help here.

**Comment by zeroSteiner** [comment]
Can you please try to create a brand new, empty campaign and export that and see if you get the same issue. That will help me determine if it's data in the campaign or the format itself.

**Comment by nullpainter** [comment]
Same issue with a brand new, empty campaign:
[Test campaign for Excel.xlsx](https://github.com/securestate/king-phisher/files/1147039/Test.campaign.for.Excel.xlsx)
![image](https://user-images.githubusercontent.com/1732347/28193932-da803c78-6894-11e7-9d12-e10ebd2e249c.png)
![image](https://user-images.githubusercontent.com/1732347/28193937-e436d70e-6894-11e7-8a1c-0271c99c293a.png)

**Comment by zeroSteiner** [comment]
Cool thanks for the details, I should have enough information to look into this now. I'll update the issue once I know more.

**Comment by zeroSteiner** [comment]
Unfortunately I don't think testing this is going to happen since I can't get access to an Office 2010 instance to use. I can't reproduce this in later versions of Office, so I'm assuming there's a compatibility issue with how the Excel file is generated. I'd recommend upgrading your version of Excel since we don't back port functionality that far.

**Comment by wolfthefallen** [comment]
I have been able to reproduce this issue using Office Version 1705(Build 8201.2193). The issue is independent of OS this is currently effecting both Linux and Windows King Phisher Clients. The primary factory I am seeing is the `AutoFilter` and `Table Form` is being removed from workbooks where their is no data in the tables. 

Primarily being seen when it is a training campaign that is being ran, and no credentials are being harvested, thus no data in tables. 

I can confirm at this time NO data is being lost during export to excel (xlsx) format. 

Error Dialog on file I was able to recreate the issue
```
Removed Feature: AutoFilter from /xl/tables/table3.xml part (Table)
Removed Feature: Table from /xl/tables/table3.xml part (Table)
```
XML out put of error from excel
```
<?xml version="1.0" encoding="UTF-8" standalone="true"?>

-<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">

<logFileName>error027440_01.xml</logFileName>

<summary>Errors were detected in file '.....xlsx'</summary>


-<removedFeatures>

<removedFeature>Removed Feature: AutoFilter from /xl/tables/table3.xml part (Table)</removedFeature>

<removedFeature>Removed Feature: Table from /xl/tables/table3.xml part (Table)</removedFeature>

</removedFeatures>

</recoveryLog>
```

**Comment by wolfthefallen** [comment]
fix in commit 4a5db85ddef344f5e135ce646205f231f939751c
